src/pages/OrderCompletePage.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { styled } from 'styled-components';
+
+const OrderCompletePage = () => {
+  const navigate = useNavigate();
+  const location = useLocation();
+  const { orderId } = location.state ?? {};
+
+  useEffect(() => {
+    if (Number.isNaN(Number(orderId))) {
```

Comment:
조건 로직이 특이한데.. 이게 맞을까요?

Reply:
location.state에 저장한 orderId값이 문자열인 줄 알아서 이런 처리를 했었는데 숫자였네요...
```suggestion
    if (orderId === undefined) {
```
이렇게 바꾸면 될 것 같습니다